persistence/src/main/proto/preferences.proto

```diff
@@ -12,5 +12,10 @@ message SpacePreference {
   optional string defaultObjectTypeKey = 1;
   repeated string pinnedObjectTypeIds = 2;
   optional string lastOpenedObject = 3;
-  optional string lastSearchQuery = 4;
+  optional GlobalSearchPreferences globalSearchPreferences = 5;
+}
+
+message GlobalSearchPreferences {
```

Comment:
`GlobalSearchHistory`

Reply:
GlobalSearchHistoryProto, because we already have GlobalSearchHistory model